src/exercism/ResistorColorDuo.java

```diff
@@ -0,0 +1,36 @@
+class ResistorColorDuo {
+    int value(String[] colors) {
+        if (colors.length < 2) {
+            throw new IllegalArgumentException("At least two colors");
+        }
+
+        int firstDigit = -1;
+        int secondDigit = -1;
+
+        String[] colorList = {"black", "brown", "red", "orange", "yellow", "green", "blue", "violet", "grey", "white"};
+        for (int i = 0; i < colorList.length; i++) {
+            if (colorList[i].equals(colors[0])) {
+                firstDigit = i;
+            }
+        }
+        for (int i = 0; i < colorList.length; i++) {
+            if (colorList[i].equals(colors[1])) {
+                secondDigit = i;
+            }
+        }
+        
+        if (firstDigit == -1) {
```

Comment:
optionally - you could add a third type of Exception if both of the names were not found